### PR TITLE
Expand main container

### PR DESCRIPTION
**Pull request by erbridge**
Please note that this needs testing before merge. I haven't been able to set up the app locally to do any testing.

### CODE REVIEW

**Comment by tahb** [comment]
I noticed there was a lot of white background at the bottom of the background on the home page when reviewing https://github.com/thedatascilab/ScenicOrNot/pull/225. I assume that's what this is trying to fix here 👍 

Here's what this branch looks like in dev, the mobile view has a slice of white still but desktop looks okay:

Screenshot 2022-08-26 at 14-29-25 ScenicOrNot

Screenshot 2022-08-26 at 14-29-37 ScenicOrNot

File: app/assets/stylesheets/generic.sass.scss [hunk 1]
```diff
@@ -56,30 +56,23 @@ dl.content {
...
   body {
     display: flex;
     flex-direction: column;
   }
 
-  #container {
-    grid-template-columns: 196px 650px;
-    background: transparent url("page-bg.png") repeat-x;
+  .main-outer-container {
     flex: 1 0 auto;
```
Comment (tahb) [inline app/assets/stylesheets/generic.sass.scss:65]: Is the fix here the way this line is applied to a new wrapping container. Beforehand was it getting confused becase #container was setting a fixed width at the same time?

**Review by tahb — APPROVED** [review]
Oh sugar I didn't mean to undraft this. With tired brain I was trying to leave a green tick that it's better than it was but might need another look for the bottom of the mobile view. 

Given that I've already made the mistake I may aswell leave the green tick that you can ignore and put it back to draft if you like!

File: app/assets/stylesheets/generic.sass.scss (hunk 1 above)
Comment (erbridge) [reply app/assets/stylesheets/generic.sass.scss:65]: The problem before was that at some point the layout had obviously gone from a single `div#container` to two nested `div.background > div#container`. There are two roles the styles were performing on the original `div#container`: styling the content of the container and styling the container itself. `body` is `display: flex`, and its child was `flex: 1 0 auto` meaning "grow me to 1 flex unit, never shrink me, and don't worry about an ideal size". With the introduction of `div.background`, we added an unstyled container between the `display: flex` and the `flex: 1 0 auto`, so `div.background` did the default behaviour for `display: flex` (no resizing), and `div#container` set an unused property `flex: 1 0 auto`. This change fixed that (while also giving more meaningful names to the selectors).

**Pull request merged**